Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/app.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <runtime>
     <!--
```

Comment:
Is this whole section (line 4-33) still needed? 
Initial comment seems out of date.
Adding a reference to empty project doesn't add this magic.

Reply:
Yes - see the review comment I added at line 37 of the new version.
I got a load more different test failures about failing to load some System.X assembly. The goal of this PR is to get the build green again, so I didn't investigate further.